static/locale/en/translations.json

```diff
@@ -200,9 +200,9 @@
   "Add a new emoji": "Add a new emoji", 
   "Change later messages to this topic": "Change later messages to this topic", 
   "Reactivate": "Reactivate", 
-  "__page_params.product_name__ Labs": "__page_params.product_name__ Labs", 
+  "Zulip Labs": "Zulip Labs", 
```

Comment:
Nope, should be "Zulip Labs".  But actually regardless, you don't need to edit the translations files at all; those are all managed via Transifex.